src/plopfile_start.js

```diff
@@ -65,6 +65,16 @@ module.exports = function (plop, cfg) {
                     force: true
                 });
             });
+            
```

Comment:
I expect to find this in the gulp task like https://github.com/swisspush/apikana/blob/develop/src/generate.js#L108.
It makes not much sense to consider samples as templates, no?